### PR TITLE
[Neutron] Add interconnection Neutron plugin

### PR DESCRIPTION
**Pull request by velp**

### CODE REVIEW

File: openstack/neutron/templates/etc/_neutron-policy.json.tpl [hunk 1]
```diff
@@ -332,7 +332,14 @@
     "update_bgpvpn_router_association": "rule:context_is_editor or rule:shared_bgpvpns",
     "delete_bgpvpn_router_association": "rule:context_is_editor or rule:shared_bgpvpns",
     "get_bgpvpn_router_association": "rule:context_is_viewer or rule:shared_bgpvpns",
-    "get_bgpvpn_router_association:tenant_id": "rule:context_is_viewer or rule:shared_bgpvpns"
-
-
+    "get_bgpvpn_router_association:tenant_id": "rule:context_is_viewer or rule:shared_bgpvpns",
```
Comment (notandy) [inline openstack/neutron/templates/etc/_neutron-policy.json.tpl:335]: If .Values.interconnection.enabled is disabled, this json list would end with a trailing comma, which is not supported by json and would probably fail to be parsed.

**Review by notandy — CHANGES_REQUESTED** [review]

File: openstack/neutron/templates/seed.yaml [hunk 2]
```diff
@@ -75,10 +75,10 @@ spec:
         role: cloud_network_admin
       - project: l3admin
         role: cloud_compute_admin
-{{- if and (.Values.interconnection.enabled) (.Values.interconnection.service_password) }}
+{{- if and (.Values.interconnection.enabled) }}
```
Comment (notandy) [inline openstack/neutron/templates/seed.yaml:78]: Probably doesn't need the and (..) prefix anymore

**Review by notandy — CHANGES_REQUESTED** [review]

File: openstack/neutron/templates/etc/_neutron-policy.json.tpl (hunk 1 above)
Comment (velp) [reply openstack/neutron/templates/etc/_neutron-policy.json.tpl:335]: Good catch, will fix.

File: openstack/neutron/templates/seed.yaml (hunk 2 above)
Comment (velp) [reply openstack/neutron/templates/seed.yaml:78]: yes, exactly

**Review by notandy — APPROVED** [review]

**Pull request merged**